src/reason_pprint_ast.ml

```diff
@@ -2218,10 +2218,9 @@ let formatAttachmentApplication finalWrapping (attachTo: (bool * layoutNode) opt
           | (hd::tl, Some (useSpace, toThis)) ->
             (* TODO: Can't attach location to this - maybe rewrite anyways *)
             let attachedArgs = makeAppList attachedList in
-            label
-              ~space:true
-              (label ~space:useSpace toThis attachedArgs)
-              wrappedListy
+              (label ~space:useSpace toThis (label
```

Comment:
The problem here is in the original code, we are making a label of 
```
Label
  left:
    Label 
      left: "let x = " 
      right: "myFunction"
  right:
    "{"
    ...
```
Once we add a comment like this: 
```
let x = 
  /* blahblahblahblahblahblahblahblahblahblah */
  myFunction {
    abc
};
```
The format tree becomes:

```
Label
  left:
    Label 
      left: "let x = " 
      right: 
        Sequence
           "/* blahblahblahblahblahblahblahblahblahblah */"
           "myFunction"
  right:
    "{"
    ...
```

In this case pretty print just formats the tree "break" everything, even when they can fit into one line:
```
let x =>
  /* exclude the scope from the hash since it is ignored by equal */
  myFunction {
    abc
  };
```